### PR TITLE
string-manipulation-2 by jaden bertino

### PR DESCRIPTION
**Pull request by jadenbertino**
image

### CODE REVIEW

File: string-manipulation-2/src/last-chars.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+/*
+  type checks
+  slice from length to str.length
+    handle if length > str.length (in that case slice from 0 not a negative number)
+*/
+
+function lastChars(length, str) {
+  if (typeof length !== 'number' || length < 0 || typeof str !== 'string') return 'invalid'
+  if (length > str.length) {
+    return str.slice(0, str.length)
```
Comment (scrumptiousdev) [inline string-manipulation-2/src/last-chars.js:12]: Or, you can just return `str`

File: string-manipulation-2/src/ransom-case.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+/*
+  split chars
+  map
+    even index -> lowercase
+    odd index -> uppercase
+  join
+*/
+
+function ransomCase(str) {
+  // readability is overrated imo
```
Comment (scrumptiousdev) [inline string-manipulation-2/src/ransom-case.js:12]: Agreed! 😭

**Review by scrumptiousdev — APPROVED** [review]
Approved!

File: string-manipulation-2/src/last-chars.js (hunk 1 above)
Comment (jadenbertino) [reply string-manipulation-2/src/last-chars.js:12]: HAHAHA I'm overcomplicating things again 😭

**Pull request merged**